tests/posix/c_lib_ext/prj.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_POSIX_API=y
+CONFIG_ZTEST=y
+
+CONFIG_POSIX_AEP_CHOICE_BASE=y
```

Comment:
I'm still kind of hesitant about having this here just because `CONFIG_POSIX_C_LIB_EXT` probably should not depend on any other POSIX options.

Will the test still pass if this line is removed?

```suggestion
```